selfdrive/pandad.py

```diff
@@ -4,10 +4,10 @@
 from panda import ensure_st_up_to_date
 
 def main(gctx=None):
-  ensure_st_up_to_date()
+  # ensure_st_up_to_date()
```

Comment:
what's this?

Reply:
This disables the Panda update, since Panda requires a patched firmware. If we let Eon reflash the Panda, it will stop working.